nengo_ocl/simulator.py

```diff
@@ -431,6 +431,15 @@ def __init__(self, network, dt=0.001, seed=None, model=None, context=None,
         self._reset_cl_rngs()
         self._probe_step_time()
 
+    @property
+    def model(self):
```

Comment:
Oh, right, but then people will get warnings if they call `nengo_ocl.Simulator.model` with an older version of Nengo.